packages/integration-tests/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint --ext .ts src tests",
     "lint:report": "pnpm lint --format json --output-file report.json",
-    "start": "jest"
```

Comment:
Is it the option that disables concurrency?

Reply:
see #1676